application/src/main/java/com/sanction/thunder/resources/UserResource.java

```diff
@@ -310,4 +324,19 @@ private Response buildResponseForDatabaseError(DatabaseError error, String email
   private boolean isValidEmail(String email) {
     return EmailValidator.getInstance().isValid(email);
   }
+
+  /**
+   * Generates a token for verifying a users email.
+   *
+   * @return 32 byte, base64 encoded random string.
+   */
+  private String generateVerificationToken() {
```

Comment:
Can we use `UUID.randomUUID().toString()` instead of `SecureRandom`? It will be unique and quicker.

Reply:
Also, SecureRandom is **NOT** unique: https://stackoverflow.com/questions/26527609/does-securerandom-guarantee-distinct-values-each-time